microshift_updating/microshift-about-updates.adoc

```diff
@@ -6,43 +6,34 @@ include::_attributes/attributes-microshift.adoc[]
+Upgrades are supported on {product-title-first} beginning with the General Availability version {ocp-version}. Supported upgrades include those from one minor version to the next in sequence, for example, from 4.14 to 4.15. Patch updates are also supported from z-stream to z-stream, for example 4.14.0 to 4.14.1.
 
 [id="microshift-about-updates-understanding-microshift-updates"]
 == Understanding {product-title} updates
-{product-title} updates are available as either RPMs or by embedding the {product-title} image in an RPM OSTree blueprint.
-You can update an {product-title} cluster by using the OpenShift CLI (`oc`).
-//Platform administrators can view new update options by looking at the output of the `oc adm upgrade` command.
-//An update begins when...
+{product-title} updates are supported on both `rpm-ostree` edge-deployed hosts and non-OSTree hosts. You can complete updates using the following methods:
 
-[NOTE]
-====
-Operators previously installed must be reinstalled using manifests.
-====
-
-[id="microshift-about-updates-rpm-updates"]
-=== RPM updates
-Using the RPM update method replaces your existing version. No rollback is possible with this update type.
-//we can call a module here or xref out; not sure the best method for our use case until we have the content
+* Embed the latest version of {product-title} in a new `rpm-ostree` system image such as {op-system-ostree-first}.
+** Only `rpm-ostree` updates include automatic rollbacks.
+** Only `rpm-ostree` updates include back ups and restores.
+* Manually update the RPMs on a non-OSTree system such as {op-system-base-full}.
 
 [id="microshift-about-updates-rpm-ostree-updates"]
 === RPM OSTree updates
-Using the RPM OSTree update path allows for system rollback.
-//we can call a module here or xref out; not sure the best method for our use case until we have the content
+Using the {op-system-ostree} `rpm-ostree` update path allows for automated backup and system rollback in case any part of the update fails. You must build a new `rpm-ostree` image and embed the new {product-title} version in that image. The `rpm-ostree` image can be the same version or an updated version, but the versions of {op-system-ostree} and {product-title} must be compatible.
+
+Check following compatibility table for details:
 
-[id="microshift-about-updates-checking-version-update-compatibility"]
-== Checking version update compatibility
-Before attempting an update, determine which version of {product-title} you have installed. Only the following update paths are supported:
+include::snippets/microshift-rhde-compatibility-table-snip.adoc[leveloffset=+2]
+
+[id="microshift-about-updates-rpm-updates"]
+=== Manual RPM updates
+Using the manual RPM update path replaces your existing version of {product-title} only. The versions of {op-system} and {product-title} must be compatible. Ensuring system health and completing additional system backups are manual processes.
```

Comment:
The amount of content that is updated really depends on how they do the upgrade. We give them the command for updating MicroShift, but there's nothing to keep them from updating all RPMs on the system. We don't necessarily need to change anything here, but if you wanted to drop that first sentence it would be OK.